editor/test/integration/save_test.clj

```diff
@@ -237,25 +237,26 @@
                                         (is (= #{:save-data :save-value} (cached-save-data-outputs "/game_object/test.go")))
                                         (is (= #{:save-data} (cached-save-data-outputs "/json/test.json"))) ; The save-value output is not :cached. Lazy loaded.
                                         (is (= #{:save-data} (cached-save-data-outputs "/script/test_module.lua"))) ; The save-value output is not :cached.
-                                        (is (= #{} (cached-save-data-outputs "/markdown/test.md"))) ; Stateless, so no save-data.
+                                        (is (= #{:save-data} (cached-save-data-outputs "/markdown/test.md")))
 
                                         (is (= #{:save-data :save-value} (cached-save-data-outputs "/added_externally.go")))
                                         (is (= #{:save-data} (cached-save-data-outputs "/added_externally.json"))) ; The save-value output is not :cached. Lazy loaded.
                                         (is (= #{:save-data} (cached-save-data-outputs "/added_externally.lua"))) ; The save-value output is not :cached.
-                                        (is (= #{} (cached-save-data-outputs "/added_externally.md"))))) ; Stateless, so no save-data.
+                                        (is (= #{:save-data} (cached-save-data-outputs "/added_externally.md")))))
 
                                     (testing "Expectations for reloaded resources."
                                       (let [external-game-object-pb-map (protobuf/read-map-without-defaults GameObject$PrototypeDesc (workspace/find-resource workspace "/game_object/test.go"))
-                                            external-lua-lines (code.util/split-lines external-lua-text)]
+                                            external-lua-lines (code.util/split-lines external-lua-text)
+                                            external-markdown-lines (code.util/split-lines external-markdown-text)]
                                         (is (check-eager-loaded-save-data! project "/game_object/test.go" external-game-object-text external-game-object-pb-map))
                                         (is (check-lazy-loaded-save-data! project "/json/test.json" external-json-text))
                                         (is (check-eager-loaded-save-data! project "/script/test_module.lua" external-lua-text external-lua-lines))
-                                        (is (check-lazy-loaded-save-data! project "/markdown/test.md" external-markdown-text))
+                                        (is (check-eager-loaded-save-data! project "/markdown/test.md" external-markdown-text external-markdown-lines))
```

Comment:
Hmm, I think we should still be able to use lazy-loading for the markdown files?

If you add `:lazy-loaded true` to the `register-code-resource-type` call, we won't load the contents of the `.md` file until the user opens it.

Currently only code resources support lazy loading. You can use as long as your node type does not need to establish connections to other nodes as a result of the file contents changing.

(We can't use it for Lua code files, for example, because they connect the referenced resources of any `require` statements as dependent build targets of themselves.)

Reply:
I tried lazy loading in my initial attempt, but couldn't update the web view. Is this because of the html output? I can give it another shot if you think should work.

Reply:
Oh, OK. It's probably fine to eager-load. Maybe we'll look into it later!